.travis.yml

```diff
@@ -1,8 +1,25 @@
 language: python
+
 python:
-    - 2.7
+    - 3.5
+
 install:
     - pip install -r requirements-dev.txt
-script: ./script/test
+
+services:
+  - postgresql
+
+addons:
+  postgresql: "9.4"  # We use 9.4 only on Travis
+
+env:
+  global:
+    - POSTGRES_USER="postgres" POSTGRESS_PASSWORD="" POSTGRES_HOST="127.0.0.1"
```

Comment:
Shouldn't we use [`DATABASE_URL`](https://github.com/kennethreitz/dj-database-url) instead?